### PR TITLE
Add PodPreset Example

### LINKED ISSUE #23: Add PodPreset Example
We're missing an example for PodPreset.

It should be possible to use an example from the Kubernetes docs.

If you want to pick up this issue, please first read the README.md and CONTRIBUTING.md. Make sure you follow the existing style and standards for other examples.

### PR DESCRIPTION
**Pull request by eliottha**
Hi there,

Not sure I did it right, but here is my attempt at adding a PodPreset example. 
If more examples are needed or I did something wrong, please let me know!

Thanks for this amazing project!

Closes #23

### CODE REVIEW

File: PodPreset/pod-preset.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: settings.k8s.io/v1alpha1
+kind: PodPreset
+metadata:
+  name: setup-database
```
Comment (ianmiell) [inline PodPreset/pod-preset.yaml:4]: Please follow the naming conventions elsewhere, ie

name: pod-preset-simple

File: PodPreset/pod-preset.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+spec:
+  selector:
+    matchLabels:
+      role: frontend
+  env:
+    - name: DB_PORT
+      value: "6379"
+  volumeMounts:
+    - mountPath: /cache
+      name: cache-volume
```
Comment (ianmiell) [inline PodPreset/pod-preset.yaml:14]: pod-preset-simple-cache-volume

File: PodPreset/pod-preset.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  volumes:
+    - name: cache-volume
```
Comment (ianmiell) [inline PodPreset/pod-preset.yaml:16]: name: pod-preset-simple-cache-volume

**Comment by eliottha** [comment]
Fixed the naming convension. Feel like an idiot for not checking that. Thanks @ianmiell

**Comment by ianmiell** [comment]
Thanks for the nudge, have merged.

**Comment by eliottha** [comment]
Thank you

**Pull request merged**